src/azure-cli/setup.py

```diff
@@ -136,7 +136,7 @@
     'pygments~=2.4',
     'pyOpenSSL>=17.1.0',
     'pytz==2019.1',
-    'pyyaml~=5.1',
+    'pyyaml~=5.2',
```

Comment:
Let's sync the version to `5.3`.

Reply:
For setup.py, we should relax the version to be the lowest workable version. Since version 5.1 through 5.1.2 have the reported critical security issue, so I think we can put 5.2.

For requiements, any workable version is fine. I think we can put the latest one for testing purpose.